extensions/amp-story/0.1/amp-story.js

```diff
@@ -91,6 +92,9 @@ const DESKTOP_HEIGHT_THRESHOLD = 550;
 /** @private @const {number} */
 const MIN_SWIPE_FOR_HINT_OVERLAY_PX = 50;
 
+/** @private @const {string} */
+const ADVERTISEMENT_ATTR_NAME = 'advertisement';
```

Comment:
nit: don't feel that strongly, but... why not just 'ad'?

Reply:
sure

Reply:
changed